pontoon/contributors/utils.py

```diff
@@ -118,14 +121,22 @@ def users_with_translations_counts(start_date=None, query_filters=None, limit=10
     # Assign properties to user objects.
     contributors = User.objects.filter(pk__in=user_stats.keys())
 
+    contributors = list(contributors)
+    if None in user_stats.keys():
+        contributors.insert(0, User(username='Imported', first_name='Imported', email='imported'))
```

Comment:
You can also simply `append()`. We typically use `insert()` if we care about the position. But here we don't, since we reorder below.

Reply:
> You probably only need `first_name` here.

If I remove username, I receive an error `Exception Type: TypeError at /sl/test-mathjazz/amo.po/authors-and-time-range/
Exception Value: Model instances without primary key value are unhashable`

If I remove email, it woudn't appear in url parameter and it's probably need to change code for using first_name instead of empty email somewhere here: 
https://github.com/mozilla/pontoon/blob/8a3e4d83150b902f3a525c998aa50443b4020b18/frontend/src/modules/search/components/SearchBox.js#L391

So, I left it as it is.